components/help/components/formatting.tsx

```diff
@@ -256,9 +256,20 @@ export default function HelpFormatting(): JSX.Element {
                 />
             </h2>
             <p>
-                <FormattedMarkdownMessage
+                <FormattedMessage
                     id='help.formatting.images.description'
-                    defaultMessage='Create in-line images using an `!` followed by the alt text in square brackets and the link in normal brackets. Add hover text by placing it in quotes after the link. See the [product documentation](!https://docs.mattermost.com/messaging/formatting-text.html#in-line-images) for details on working with in-line images.'
+                    defaultMessage='Create in-line images using an `!` followed by the alt text in square brackets and the link in normal brackets. Add hover text by placing it in quotes after the link. See the <link>product documentation</link> for details on working with in-line images.'
```

Comment:
Technically, the backticks around the exclamation mark are Markdown, but I think this is fine to leave as-is since they're still wrapping the important character. These pages are going to be removed at some time in the next few months anyway